src/helpers/useFallbackImage.ts

```diff
@@ -0,0 +1,35 @@
+import { SyntheticEvent, useCallback, useEffect, useRef } from "react"
+
+export function useFallbackImage(fallbackSrc: string) {
+  const imgParentRef = useRef<HTMLDivElement>(null)
+  const imgRefs = useRef<(HTMLImageElement | null)[]>([])
+
+  const onError = useCallback(
+    (e: SyntheticEvent<HTMLImageElement>) => {
+      e.currentTarget.src = fallbackSrc
+    },
+    [fallbackSrc]
+  )
+
+  useEffect(() => {
+    if (imgParentRef && imgParentRef.current) {
+      const numberOfImages = imgParentRef.current.children.length
+      for (let i = 0; i < numberOfImages; i++) {
```

Comment:
Is there a way you have been testing with more than one image in an ImageCard?

Reply:
i tested it for example in `Test: Coliseum` listing by replacing images prop with sth like this (`listingView.tsx` line 511):
```
images={imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize)).map(
            (imageUrl: string, index) => {
              return {
                url: index % 2 ? imageUrl : "test",
              }
            }
          )}
```